### ISSUE TITLE
Deploy/CI to Cloud Foundry?

### ISSUE THREAD

**Issue by jeremiak** [18F/federalist#38]
This might be super premature, so please push back if it is. How would you all feel about setting up continuous deployment so that master is always live on a Cloud Foundry app. That way it might be easier to show people and get feedback?

Also, if not now would this "feature" slot more nicely into the second or third phase?

**Comment by dhcole** [comment]
I was thinking about this for next sprint, based on all the questions we get. We need to take care of some security tasks first though before we can do that. For instance, we're just running Jekyll with whatever plugin people have in the repo, which could be malicious. And we're not checking to make sure only 18F members can use it.

Let's keep open and discuss tomorrow when we plan out our next sprint.

**Comment by jeremiak** [comment]
:+1: ^^

**Comment by dhcole** [comment]
Moving forward in #51
